plugin/sender-constrained-token-plugin.lua

```diff
@@ -65,12 +65,12 @@ function _M.execute(config)
     end
 
     local auth_header = ngx.req.get_headers()['Authorization']
-    if auth_header and string.len(auth_header) > 7 and string.lower(string.sub(auth_header, 1, 7)) == 'bearer ' then
+    if auth_header and string.len(auth_header) > 7 and string.sub(auth_header, 1, 7) == 'Bearer ' then
```

Comment:
If I remember correctly HTTP2 uses lowercase `bearer`, or maybe I've seen it changed to lowercase by a load balancer (like HA-proxy). I think it's safer to leave that lowercase comparison.